README_landice.md

```diff
@@ -2,17 +2,29 @@
 
 ## COMPASS conda environment
 
-To set up and run landice test cases from COMPASS, you will need a conda
+To set up and run ocean test cases from COMPASS, you will need a conda
```

Comment:
This inadvertent change should be removed.

Reply:
Bad copy/paste job, sorry!